<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>640</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Load Actions</string>
  </property>
  <widget class="QListWidget" name="allAct">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>265</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>265</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DropOnly</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="taCombo">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>236</y>
     <width>197</width>
     <height>29</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>239</y>
     <width>98</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Agent :</string>
   </property>
  </widget>
  <widget class="QListWidget" name="agAct">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>282</y>
     <width>265</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>265</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="dropBtn">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>190</y>
     <width>338</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Drop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>440</y>
     <width>338</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Actions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refBtn">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>236</y>
     <width>29</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../opt/Basefount/Miarmy/maya/icons/ref.png</normaloff>../../../../../../opt/Basefount/Miarmy/maya/icons/ref.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="impBtn">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>10</y>
     <width>67</width>
     <height>83</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>99</y>
     <width>67</width>
     <height>82</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delAgBtn">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>280</y>
     <width>67</width>
     <height>153</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>549</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Miarmy Contents Copy</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>550</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>576</y>
     <width>322</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QCheckBox" name="acChk">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Action</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="trChk">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Transition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deChk">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Decision</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pickBtn">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Pick</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="assignBtn">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>92</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Assign</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>485</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>485</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>484</y>
     <width>222</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Action register to stateNode</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>510</y>
     <width>300</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QComboBox" name="nodeCmb">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="refrBtn">
      <property name="maximumSize">
       <size>
        <width>26</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../../opt/Basefount/Miarmy/maya/icons/ref.png</normaloff>../../../../../../opt/Basefount/Miarmy/maya/icons/ref.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="raBtn">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Register Actions</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
